test/unit/data-generator.js

```diff
@@ -0,0 +1,361 @@
+var expect = require('chai').expect,
+    objectAssign = require('object-assign'),
+    generate = require('../../lib/data-generator.js'),
+    EventEmitter = require('events').EventEmitter;
+
+var root = {
+      title: ''
+    },
+    suite = {
+      title: 'suite'
+    },
+    test = {
+      title: 'test',
+      ctx: {},
+      err: 'error'
```

Comment:
This seems like a failed test, so the variable should be named accordingly. Also, you have a bunch of values here that are for your input and a bunch that are for your output (expected). You should name the accordingly and group them separately.